controllers/machinepool.go

```diff
@@ -61,7 +55,7 @@ func (r *GarbageCollectorController) Reconcile(ctx context.Context, req ctrl.Req
 		return r.reconcileDelete(ctx, &machinePool, logger)
 	}
 
-	return ctrl.Result{Requeue: true, RequeueAfter: time.Minute * 5}, nil
+	return ctrl.Result{}, nil
```

Comment:
This was requeing `MachinePools` that were not being deleted. We are not interested in reconciling those CRs. Should a `MachinePool` be deleted, the API will queue it for us.